test/rulesets/Base/fastmath_able.jl

```diff
@@ -131,13 +131,25 @@ const FASTABLE_AST = quote
             rrule_test(f, Δz, (x, x̄), (y, ȳ))
         end
 
-        @testset "$f(x::$T, y::$T)" for f in (/, +, -, hypot), T in (Float64, ComplexF64)
+        @testset "$f(x::$T, y::$T)" for f in (/, +, -, hypot), T in (Float32, Float64, ComplexF64)
             x, Δx, x̄ = 10rand(T, 3)
             y, Δy, ȳ = rand(T, 3)
             Δz = randn(typeof(f(x, y)))
 
-            frule_test(f, (x, Δx), (y, Δy))
-            rrule_test(f, Δz, (x, x̄), (y, ȳ))
+            if T != Float32
```

Comment:
had to exclude this case because test tolerance is too strict for Float32

Reply:
Could you decrease the tolerance using the keyword arguments of the testers?
e.g. something like `frule_test(f, (x, Δx), (y, Δy); atol=0, rtol = sqrt(eps(T)))`